bungeecord/src/main/java/net/william278/huskchat/bungeecord/event/BaseEvent.java

```diff
@@ -0,0 +1,16 @@
+package net.william278.huskchat.bungeecord.event;
+
+import net.md_5.bungee.api.plugin.Cancellable;
+import net.md_5.bungee.api.plugin.Event;
+
+public class BaseEvent extends Event implements Cancellable {
```

Comment:
I think BaseEvent should be --> to BungeeEvent; likewise for VelocityEvent, just to disambiguate here